pkg/handler/composer.go

```diff
@@ -14,6 +14,8 @@ type StoreComposer struct {
 	Concater           ConcaterDataStore
 	UsesLengthDeferrer bool
 	LengthDeferrer     LengthDeferrerDataStore
+	Server             ServerDataStore
```

Comment:
As mentioned in https://github.com/tus/tusd/issues/1064#issuecomment-2401568772, I am not sure on my own whether `Server` is a good name as it does have a different meaning in the HTTP-land. How about `ContentServer` as an alternative?

We would then have `StoreComposer.ContentServer`, `StoreComposer.UsesContentServer`, `StoreComposer.UseContentServer`, and `tusd.ContentServerDataStore`.

Reply:
works for me, I don't really care :P.